### ISSUE TITLE
Missing boundary header

### ISSUE THREAD

**Issue by Anurag--Singh** [hypertrack/hyperlog-android#11]
I get the following error when trying to push the logs to server using:

`HyperLog.pushLogs`

`onError :: called with :: hlErrorResponse.getErrorCode = [400], hlErrorResponse.getErrorMessage = [400: The request could not be understood by the server due to malformed syntax. Missing boundary header]`

**Comment by Aravindaful** [comment]
did you resolve this problem?

**Comment by Anurag--Singh** [comment]
@Aravindaful No, I couldn't directly. I modified the source code as per my need and removed replaced volley with retrofit.
